### DISCUSSION TITLE
Any plans for web components version ?

### DISCUSSION THREAD

**Discussion by ghost** [primer/react#3211] [Ideas]
I'm not sure where to ask this question, so I apologize if this is the wrong place. 

**I'm wondering if there are any plans to make a [web components](https://developer.mozilla.org/en-US/docs/Web/API/Web_components)  version of this project.**

I'm very interested in using web components for my own development and I think it would be great to have a compatible version of this project.

**Comment by lesliecdubs** [comment]
Hi @erDaren 👋🏻 We do not have plans to offer the Primer React component set via web components. 

That said, the Primer team also maintains [Catalyst](https://catalyst.rocks/), a set of patterns and techniques for developing web components, which may be of interest to you.
